ngraph_bridge/enable_variable_ops/ngraph_capture_variables.cc

```diff
@@ -105,6 +187,14 @@ Status CaptureVariables(Graph* graph, std::set<string> skip_these_nodes) {
     NGRAPH_VLOG(4) << "Removing: " << node->name();
     graph->RemoveNode(node);
   }
+
+  if (std::getenv(NGraphPrefetchSharedResouce::NGRAPH_TF_USE_PREFETCH) !=
+      nullptr) {
+    if (prefetch_node != nullptr) {
+      return ReplacePrefetch(graph, prefetch_node);
+    }
```

Comment:
Should we add a VLOG in the else  of if (prefetch_node != nullptr) {
saying "prefetch_node = nullptr", No prefetch op created.....
something like this, so atleast we know.

Reply:
Good idea. There is no else, and the else will be created only for VLOG. I can add VLOG saying replacing this prefetch op with NGraphPrefetch

Reply:
That works too.